pkg/clusteragent/clusterchecks/dispatcher_main.go

```diff
@@ -68,6 +81,11 @@ func (d *dispatcher) Unschedule(configs []integration.Config) {
 		if !c.ClusterCheck {
 			continue // Ignore non cluster-check configs
 		}
+		if isServiceCheck(c) {
```

Comment:
Do we ever get to this block for service checks since they have ClusterCheck set to false?